### PR TITLE
Service Fabric Migration

### PR DESCRIPTION
**Pull request by mwesigwaguma**
## Description

Migrating from .Net sdk dependency to autorest generated sdk. Since Powershell does not support Track 2 .Net sdk , we have decided to migrate autorest.powershell generated sdk.
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by VeryEarly** [comment]
What is the purpose of this PR? What are differences from: https://github.com/Azure/azure-powershell/pull/22860

**Comment by mwesigwaguma** [comment]
> What is the purpose of this PR? What are differences from: #22860

They are two different sdks. Service Fabric and Service Fabric Managed Clusters
If you look in https://github.com/mwesigwaguma/azure-sdk-for-net/tree/main/sdk, you will find both Service Fabric and Service Fabric Managed Clusters. I am migrating both from .Net dependency

**Comment by mwesigwaguma** [comment]
> What is the purpose of this PR? What are differences from: #22860

Can I get this merged if nothing else

File: src/ServiceFabric/ServiceFabric.Management.sdk/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+[assembly: Guid("B965F0A8-D610-4275-B5EA-8F2D806F337C")]
+[assembly: AssemblyVersion("3.2.0")]
+[assembly: AssemblyFileVersion("3.2.0")]
+
+[assembly: AssemblyTitle("Microsoft Azure Powershell - Service Fabric")]
+[assembly: AssemblyCompany(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCompany)]
+[assembly: AssemblyProduct(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyProduct)]
+[assembly: AssemblyCopyright(Microsoft.WindowsAzure.Commands.Common.AzurePowerShell.AssemblyCopyright)]
+[assembly: InternalsVisibleTo("Microsoft.Azure.PowerShell.Cmdlets.ServiceFabric.Test")]
```
Comment (VeryEarly) [inline src/ServiceFabric/ServiceFabric.Management.sdk/Properties/AssemblyInfo.cs:28]: remove this line

File: src/ServiceFabric/ServiceFabric/ServiceFabric.csproj [hunk 2]
```diff
@@ -14,11 +14,11 @@
...
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.1" />
     <PackageReference Include="Microsoft.Azure.KeyVault.WebKey" Version="3.0.1" />
-	<PackageReference Include="Microsoft.Azure.Management.ServiceFabric" Version="1.3.0" />
   </ItemGroup>
 
-	<ItemGroup>
-		<ProjectReference Include="..\ServiceFabricManagedClusters.Management.Sdk\ServiceFabricManagedClusters.Management.sdk.csproj" />
+  <ItemGroup>
+	<ProjectReference Include="..\ServiceFabricManagedClusters.Management.Sdk\ServiceFabricManagedClusters.Management.sdk.csproj" />
+    <ProjectReference Include="..\ServiceFabric.Management.Sdk\ServiceFabric.Management.sdk.csproj" />
```
Comment (VeryEarly) [inline src/ServiceFabric/ServiceFabric/ServiceFabric.csproj:21]: indentation

**Review by VeryEarly — CHANGES_REQUESTED** [review]

File: src/ServiceFabric/ServiceFabric/ServiceFabric.csproj (hunk 2 above)
Comment (mwesigwaguma) [reply src/ServiceFabric/ServiceFabric/ServiceFabric.csproj:21]: updated

File: src/ServiceFabric/ServiceFabric.Management.sdk/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (mwesigwaguma) [reply src/ServiceFabric/ServiceFabric.Management.sdk/Properties/AssemblyInfo.cs]: updated

**Review by VeryEarly — APPROVED** [review]

**Review by VeryEarly — APPROVED** [review]

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**